### PR TITLE
Aligned text in feature list for mobiles

### LINKED ISSUE #5651: Text alignment issue in the feature list
### Description
The text alignment in the feature list section on the mobile view doesn't look good, especially after the first two lines. This misalignment affects readability and overall user experience.

### URL
The issue can be observed on the [Layer5 Pricing page](https://layer5.io/pricing).

### Steps to Reproduce
1. Open the Layer5 Pricing page on a mobile device.
2. Navigate to the feature list section.
3. Observe the text alignment issues

### Expected Behavior
The text should be properly aligned, maintaining a consistent format for better readability.

### Screenshots
Screenshot_2024-07-08-17-26-27-251_com android chrome

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Harsh-svg988**
**Description**

This PR fixes #5651

**Notes for Reviewers**
I Made the content mobile friendly by applying some css . After two lines it was misaligning which is now fixed and increases overall experience to user

Screenshot 2024-07-10 at 12 16 32 AM

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3fa111791045108d1f0f17797db2f4ede825e7b3 at: https://668d8a94d87737d0a9e5297a--layer5.netlify.app

File: src/components/PlanCard/collapsible-details.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const FeatureDetails = (props) => {
 
   return (
     <FeatureDetailsWrapper>
-      <div onClick={function () {
+      <div  onClick={function () {
```
Comment (Ashparshp) [inline src/components/PlanCard/collapsible-details.js:53]: Please do not change anything else in the code. Specifically, avoid making whitespace changes or other lint-related modifications.

File: src/sections/Pricing/data.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const details = [
         enterprise: <GiCheckMark className="yes-icon" />,
       },
       {
-        feature: "Configurable OAuth callbacks",
+        feature: "Configurable OAuth callbacks ",
```
Comment (Ashparshp) [inline src/sections/Pricing/data.js:28]: Please do not change anything else in the code. Specifically, avoid making whitespace changes or other lint-related modifications.

**Review by Ashparshp — COMMENTED** [review]
LGTM. Thanks for the changes and the fix and please avoid making any additional modifications..

**Comment by l5io** [comment]
🚀 Preview for commit f8ff2b4b23bc4509bdac119029140710f4877de4 at: https://668e7cd74e75530cf9954be1--layer5.netlify.app

**Comment by Harsh-svg988** [comment]
will it get merged or should i have to update @Ashparshp

**Comment by l5io** [comment]
🚀 Preview for commit ad3c7dac6423ef32c037aa8d21bd76e568545c30 at: https://668ed2d9330b7a3493169d8d--layer5.netlify.app

**Comment by Ashparshp** [comment]
@Harsh-svg988 Ain't you creating new branches for new changes?

**Comment by Ashparshp** [comment]
@Harsh-svg988 Also, please do mention the issue in PR!! So, that it may link with that only!

**Pull request closed without merging**